tests/client/test_botx_method_callback.py

```diff
@@ -441,3 +442,57 @@ async def test__botx_method_callback__callback_successful_received(
     # - Assert -
     assert await task == UUID("21a9ec9e-f21f-4406-ac44-1a78d2ccf9e3")
     assert endpoint.called
+
+
+async def test__botx_method_callback__bot_wait_callback(
+    respx_mock: MockRouter,
+    httpx_client: httpx.AsyncClient,
+    host: str,
+    bot_id: UUID,
+    bot_account: BotAccountWithSecret,
+) -> None:
+    # - Arrange -
+    endpoint = respx_mock.post(
+        f"https://{host}/foo/bar",
+        json={"baz": 1},
+        headers={"Content-Type": "application/json"},
+    ).mock(
+        return_value=httpx.Response(
+            HTTPStatus.ACCEPTED,
+            json={
+                "status": "ok",
+                "result": {"sync_id": "21a9ec9e-f21f-4406-ac44-1a78d2ccf9e3"},
+            },
+        ),
+    )
+    built_bot = Bot(
+        collectors=[HandlerCollector()],
+        bot_accounts=[bot_account],
+        httpx_client=httpx_client,
+    )
+
+    built_bot.call_foo_bar = types.MethodType(call_foo_bar, built_bot)
+
+    # - Act -
+    async with lifespan_wrapper(built_bot) as bot:
+        foo_bar = await bot.call_foo_bar(bot_id, baz=1, wait_callback=False)
+        task = asyncio.create_task(bot.wait_botx_method_callback(foo_bar, None))
+        await asyncio.sleep(0)
```

Comment:
Add `  # Return control to event loop` comment

Reply:
added